app/src/components/v-upload.vue

```diff
@@ -86,9 +121,7 @@ function useUpload() {
 		};
 
 		try {
-			if (!validFiles(files)) {
-				throw new Error('An error has occurred while uploading the files.');
-			}
+			if (!validFiles(files)) return;
```

Comment:
Does this now silently ignore?

Reply:
The error is not handled in `validFiles` function

Reply:
Ah you're doing the notifications in validFiles, gotcha